src/ansible_navigator/utils/serialize_defs.py

```diff
@@ -0,0 +1,10 @@
+"""Definitions related to content serialization."""
```

Comment:
Out of curiosity, is this needed in its own file vs. having it in ./serialize.py?

Reply:
Ah, I found the answer to my question immediately. I see the import in other files. Carry on.

Reply:
_This_ is a great question, which is where I need a little help.  The definition was moved out of serialize.py, as part of an effort to avoid the circular import when type_checking......, although it didn't eliminate it I left it there.

I don't think the TYPE_CHECKING condition is necessarily a bad thing, but it does make me wonder if there isn't a better way to structure everything....

@relrod @alisonlhart If you have any ideas about a better layout, I'm listening...